lib/tasks/stop_data_export.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops do
+  task export_old_data: :environment do
+    CSV.open('old_stop_data.csv', 'w') do |csv|
+      headers = %w[id has_power shared_sign_post system_map_exists trash]
```

Comment:
So is *this* a case where we could use `write_headers: true` and then shovel hashes of attributes to the CSV rather than arrays?

Reply:
That sounds, right but I'm having trouble figuring out what this would look like. Can you with an example? 
I tried something like this but only to get a no method error.
`csv << { id: stop.id, has_power: stop.has_power,`
                ` shared_sign_post: stop.shared_sign_post,`
                 `system_map_exists: stop.system_map_exists,`
                 `trash: stop.trash }`

Reply:
I'm going to change my suggestion to be that we specify the headers in the argument to `open`, and use `write_headers: true`, now that I've actually read the docs. https://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html#method-c-new

But it seems you're correct, it'll have to be an array.